cinema/views.py

```diff
@@ -43,10 +45,42 @@
 
         return MovieSerializer
 
+    def get_queryset(self):
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+        queryset = self.queryset
+
+        if actors:
+            actors_ids = [int(str_id) for str_id in actors.split(",")]
+            queryset = Movie.objects.filter(actors__id__in=actors_ids)
+
+        if genres:
+            genres_ids = [int(str_id) for str_id in genres.split(",")]
+            queryset = Movie.objects.filter(genres__id__in=genres_ids)
+
+        if title:
+            queryset = Movie.objects.filter(title__icontains=title)
```

Comment:
The filtering logic for actors, genres, and title should be combined using the `queryset` variable instead of reassigning it to a new query. This ensures that all filters are applied cumulatively.